tests/scripts/single_card/run_demos_single_card_n300_func_tests.sh

```diff
@@ -0,0 +1,15 @@
+#/bin/bash
+
+set -eo pipefail
+
+if [[ -z "$TT_METAL_HOME" ]]; then
+  echo "Must provide TT_METAL_HOME in environment" 1>&2
+  exit 1
+fi
+
+source tests/scripts/single_card/run_demos_single_card_wh_common.sh
+
+# Not working on N150, working on N300
+unset WH_ARCH_YAML
+rm -rf built
```

Comment:
why do we need to remove the built folder here?

Reply:
I'm not sure about that, it was there from before and I didn't modify it.

Reply:
can we see if this still works without this?

Reply:
let's try this experiment before checking in

Reply:
Removed that line and re-tested, passed